src/Auth0Client.ts

```diff
@@ -668,8 +668,17 @@ export default class Auth0Client {
       throw new Error('Invalid state');
     }
 
+    const {
+      scope,
+      audience,
+      redirect_uri,
+      ignoreCache,
```

Comment:
```suggestion
      timeoutInSeconds,
      ignoreCache,
```

Reply:
Implemented this manually rather than accepting the solution.